accepted/PSR-13-links.md

```diff
@@ -4,7 +4,7 @@ Hypermedia links are becoming an increasingly important part of the web, in both
 and various API format contexts. However, there is no single common hypermedia format, nor
 is there a common way to represent links between formats.
 
-This specification aims to provide PHP developers with a simple, common way of representing a
+This specification aims at providing PHP developers with a simple, common way of representing a
```

Comment:
This change is incorrect.  "aims to provide" is valid English grammar.  "aims at providing" I've never seen or heard.

Reply:
Well both coexist according to what I find online. I've always seen this version more than the other one. I guess I'll remove that part! :)

Reply:
I'd agree, the original reads correctly